package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-map-gl",
   "description": "A React wrapper for MapboxGL-js and overlay API.",
-  "version": "1.7.2",
+  "version": "1.7.3",
```

Comment:
I think the version needs to bump past the two "broken" versions that were made outside of master, so change this to 1.8.2 or 1.9 (not needing babel-polyfill is kind of a new feature).